### PR TITLE
feat(render): language support with `hl`

### PR DESCRIPTION
**Pull request by zhuangya**
ref #16

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -58,7 +59,8 @@ class Reaptcha extends Component<Props> {
     tabindex: 0,
     explicit: false,
     inject: true,
-    isolated: false
+    isolated: false,
+    hl: 'en'
```
Comment (sarneeh) [inline lib/index.js]: The default value should be unspecified for auto-detection.

File: lib/index.test.js [hunk 2]
```diff
@@ -129,7 +130,46 @@ test('should render invisible recaptcha', t => {
...
+      onVerify={onVerify}
+      onExpire={onExpire}
+      onError={onError}
+      explicit={false}
+    />
+  );
+
+  t.true(renderSpy.calledOnce);
+  t.true(
+    renderSpy.calledWith(sinon.match.any, {
```
Comment (sarneeh) [inline lib/index.test.js]: This test checks if recaptcha renders with the `hl` property. All other properties are unnecessary. Try this:
```
t.plan(1);
(...)
t.true(renderSpy.calledWith(sinon.match.any, sinon.match({ hl: 'fr' }));

File: lib/index.test.js [hunk 3]
```diff
@@ -129,7 +130,46 @@ test('should render invisible recaptcha', t => {
...
+  const onVerify = sinon.stub();
+  const onExpire = sinon.stub();
+  const onError = sinon.stub();
+
+  mount(
+    <Reaptcha
+      sitekey="my-key"
+      theme="dark"
+      size="normal"
+      hl='fr'
```
Comment (sarneeh) [inline lib/index.test.js]: Double quotes here.

File: lib/index.test.js [hunk 4]
```diff
@@ -129,7 +130,46 @@ test('should render invisible recaptcha', t => {
...
+      hl: null
+    })
+  );
+});
+
+test('should render recaptcha with hl', t => {
+  t.plan(2);
+
+  const onVerify = sinon.stub();
+  const onExpire = sinon.stub();
```
Comment (sarneeh) [inline lib/index.test.js]: These are also optional props so they are totally redundant in this test.

File: lib/index.test.js [hunk 5]
```diff
@@ -92,7 +92,8 @@ test('should render recaptcha', t => {
       callback: onVerify,
       'expired-callback': onExpire,
       'error-callback': onError,
-      isolated: null
+      isolated: null,
+      hl: 'en'
```
Comment (sarneeh) [inline lib/index.test.js]: Change the default.

File: lib/index.js [hunk 6]
```diff
@@ -58,7 +60,8 @@ class Reaptcha extends Component<Props> {
     tabindex: 0,
     explicit: false,
     inject: true,
-    isolated: false
+    isolated: false,
+    hl: ''
```
Comment (zhuangya) [inline lib/index.js]: i'm not very familiar with `flow`, should this be `''` or `undefined`?
not sure about this 😵

File: lib/index.test.js [hunk 7]
```diff
@@ -129,11 +130,29 @@ test('should render invisible recaptcha', t => {
...
+  t.plan(1);
+
+  mount(
+    <Reaptcha
+      sitekey="my-key"
+      hl="fr"
+    />
+  );
+
+  t.true(
```
Comment (zhuangya) [inline lib/index.test.js]: does this test case make sense?

File: lib/index.test.js [hunk 8]
```diff
@@ -129,11 +130,29 @@ test('should render invisible recaptcha', t => {
...
+      isolated: true,
+      hl: null
     })
   );
 });
 
+test('should render recaptcha with hl', t => {
+  t.plan(1);
+
+  mount(
```
Comment (sarneeh) [inline lib/index.test.js]: Make it like the other tests:
`<Reaptcha {...defaultProps} hl="fr" />`

File: lib/index.test.js (hunk 7 above)
Comment (sarneeh) [reply lib/index.test.js]: Yes, this test is cool now :) Just change the the above comment please :)

File: lib/index.js (hunk 6 above)
Comment (sarneeh) [reply lib/index.js]: Can be both, but as I've made the `className` and `id` with `''` - keep it the same way.

File: README.md [hunk 9]
```diff
@@ -242,11 +242,12 @@ Available and usable `Reaptcha` instance methods:
 | onVerify  | **yes**  | `Function`                                  | -               | Callback function executed on user's captcha verification. Returns [user response token](https://developers.google.com/recaptcha/docs/verify) |
 | onExpire  | no       | `Function`                                  | -               | Callback function executed when the reCAPTCHA response expires and the user needs to re-verify                                                |
 | onError   | no       | `Function`                                  | -               | Callback function executed when reCAPTCHA fails with an error                                                                                 |
+| hl        | no       | `string`                                    | ''              | Language code for reCAPTCHA, available [language codes](https://developers.google.com/recaptcha/docs/language)                                |
```
Comment (sarneeh) [inline README.md]: Please change it to: 
[Language code](https://developers.google.com/recaptcha/docs/language) for reCAPTCHA

**Comment by zhuangya** [comment]
job's done.

**Pull request merged**